HashRegistrarSimplified.sol

```diff
@@ -334,11 +334,11 @@ contract Registrar {
         } else if(auctionState != Mode.Reveal) {
             // Invalid phase
             throw;
-        } else if (_value < minPrice) {
+        } else if (actualValue < minPrice) {
```

Comment:
Actually... Why do we allow making a bid, which supposedly contains `_value` - the 'true' value, but then allow the actual bid value to be something else? It makes no sense (imho) to have such 'malleability'. I mean, there is a salt to provide entropy, so imho, the `_value` should be the sole source of **what was actually the bid value**. 

Therefore, I suggest that instead of 
```
uint actualValue = min(_value, bid.value())
```
We instead verify that `bid.value() >= _value`. And if not, consider the `bid` invalid. 

It's not a big issue, but I don't really like the ambiguity that is there right now.